tests/test_bookmarks.py

```diff
@@ -177,7 +176,7 @@ def test_run(self):
             first_rec = first_sync_records.get(stream).get('messages')[0].get('data')
             first_rec_id = first_rec.get('id')
             first_rec_version = first_rec.get('version')
-            updated_record = self.client.update(stream, first_rec_id, first_rec_version)
+            updated_record = self.client.update(stream, first_rec_id, first_rec_version, obj=first_rec)
```

Comment:
`obj` is too generic. Can we rename it to something else? Even `stream_data` seems better to me.